frontend/projects/upgrade/src/app/shared/components/mat-confirm-dialog/mat-confirm-dialog.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MatConfirmDialogComponent } from './mat-confirm-dialog.component';
+
+describe('MatConfirmDialogComponent', () => {
```

Comment:
if we aren't going to write tests, we should either delete this file or mark it `xdescribe` for now